ibc/lightclients/groth16/client_state.go

```diff
@@ -51,13 +53,14 @@ type MptProof struct {
 var _ exported.ClientState = (*ClientState)(nil)
 
 // NewClientState creates a new ClientState instance.
-func NewClientState(latestHeight uint64, stateTransitionVerifierKey []byte, stateMembershipVerifierKey []byte, codeCommitment []byte, genesisStateRoot []byte) *ClientState {
+func NewClientState(latestHeight uint64, stateTransitionVerifierKey string, stateMembershipVerifierKey []byte, groth16Vk []byte, codeCommitment []byte, genesisStateRoot []byte) *ClientState {
 	return &ClientState{
 		LatestHeight:               latestHeight,
 		CodeCommitment:             codeCommitment,
 		GenesisStateRoot:           genesisStateRoot,
-		StateTransitionVerifierKey: stateMembershipVerifierKey,
+		StateTransitionVerifierKey: stateTransitionVerifierKey,
 		StateMembershipVerifierKey: stateMembershipVerifierKey,
+		Groth16Vk:                  groth16Vk,
```

Comment:
[nit] the other fields in this struct use the suffix `VerifierKey` so unless there's a strong reason to do so, can we conform to that convention?

```suggestion
		Groth16VereifierKey:                  groth16Vk,
```